### ISSUE TITLE
Torrent

### ISSUE THREAD

**Issue by Ashik2233** [subhra74/xdm#610]
## Checklist

- [ ] I'm reporting a feature request
- [ ] I've verified that I'm running xdm latest version
- [ ] I've searched the bugtracker for similar feature requests including closed ones

## Description

WRITE DESCRIPTION HERE
I can't download torrent via xtreme download manager

**Comment by milldogtjm** [comment]
Here is the programs description again for you"Xtreme Download Manager (XDM) is a powerful tool to increase download speeds up to 500%, save streaming videos from YouTube, DailyMotion, Facebook, Vimeo, Google Video and 1000+ other websites, resume broken/dead downloads, schedule and convert downloads. XDM seamlessly integrates with Google Chrome, Mozilla Firefox Quantum, Opera, Vivaldi and other Chroumium and Firefox based browsers, to take over downloads and to allow you to save almost all streaming videos from the web. XDM has a built in video converter which lets you convert your downloaded videos to different formats so that you can watch them on your mobile or TV (100+ devices are supported)
Basically this Download manager / video converter isn't a replacement for downloading torrent, magnet links type files, sorry but you will still need a torrent files and magnet links downloader like these top 5 as of 2021:     qBittorrent
    Vuze
    Deluge
    uTorrent
    BitTorrent

**Comment by hemish04082005** [comment]
@Ashik2233 Actually, free download manager has a torrent support and can convert files after downloading, can download youtube and other sites videos, and is not based on Java thus reducing dependencies.
Also, Free download manager is open-source and cross-platform

**Comment by ahmed-tasaly** [comment]
> @Ashik2233 Actually, free download manager has a torrent support and can convert files after downloading, can download youtube and other sites videos, and is not based on Java thus reducing dependencies.
> Also, Free download manager is open-source and cross-platform
@hemish04082005 

FreeDownloadManager after version 5 is no longer opensource

**Comment by Continous** [comment]
XDM is intended to be a download manager. I think it is reasonable that a user may expect that it would support arguably **the** most popular download protocol on the internet, bittorrent.
